src/servers/Http1Server.cc

```diff
@@ -234,8 +234,6 @@ Http::One::Server::processParsedRequest(Http::StreamPointer &context)
         if (!supportedExpect) {
             clientStreamNode *node = context->getClientReplyContext();
             quitAfterError(request.getRaw());
-            // setLogUri should called before repContext->setReplyToError
-            setLogUri(http, urlCanonicalClean(request.getRaw()));
```

Comment:
Should we add a "setReplyToError() requires log_uri" assertion here like we did in another place where a setLogUri() call was removed?